File: problemas/1002/AreaCirculo.cpp
#include <iostream>
#include <cmath>
 
using namespace std;
 
int main() {
    double raio = 0;
    double pi = 3.14159;
    
    scanf("%lf", &raio);
    
    printf("A=%.4lf\n", pi * pow(raio,2));
 
    return 0;
}

File: solucoes/1002/AreaCirculo.dart
import 'dart:io';
import 'dart:math';

void main() {
    double pi = 3.14159;
    double? raio = double.parse(stdin.readLineSync()!);
    double area = pi*pow(raio,2);
    var ans = area.toStringAsFixed(4);

    print('A=$ans');
}